EMOBON00092/manual_example_ro-crate-metadata.json

```diff
@@ -54,174 +41,216 @@
         "license": {"@id": "Creative Commons (CC-BY 4.0)"},                      
         "publisher": {"@id": "https://ror.org/0038zss60"},                          
         "hasPart": [
-                        {"@id": "https://github.com/emo-bon/mgf-s3-prototype/blob/main/EMOBON00092/DBH.merged.cmsearch.all.tblout.deoverlapped.dvc"},
-                        {"@id": "https://github.com/emo-bon/mgf-s3-prototype/blob/main/EMOBON00092/DBH.merged.fasta.dvc"},
-                        {"@id": "https://github.com/emo-bon/mgf-s3-prototype/blob/main/EMOBON00092/DBH.merged.motus.tsv.dvc"},
-                        {"@id": "https://github.com/emo-bon/mgf-s3-prototype/blob/main/EMOBON00092/DBH.merged.qc_summary.dvc"},
-                        {"@id": "https://github.com/emo-bon/mgf-s3-prototype/blob/main/EMOBON00092/DBH.merged.unfiltered_fasta.dvc"},
-                        {"@id": "https://github.com/emo-bon/mgf-s3-prototype/blob/main/EMOBON00092/DBH.merged_CDS.faa.dvc"},
-                        {"@id": "https://github.com/emo-bon/mgf-s3-prototype/blob/main/EMOBON00092/DBH.merged_CDS.ffn.dvc"},
-                        {"@id": "https://github.com/emo-bon/mgf-s3-prototype/blob/main/EMOBON00092/DBH_1_clean.fastq.trimmed.fasta.dvc"},
-                        {"@id": "https://github.com/emo-bon/mgf-s3-prototype/blob/main/EMOBON00092/DBH_1_clean.fastq.trimmed.qc_summary.dvc"},
-                        {"@id": "https://github.com/emo-bon/mgf-s3-prototype/blob/main/EMOBON00092/DBH_2_clean.fastq.trimmed.fasta.dvc"},
-                        {"@id": "https://github.com/emo-bon/mgf-s3-prototype/blob/main/EMOBON00092/DBH_2_clean.fastq.trimmed.qc_summary.dvc"},
-                        {"@id": "https://github.com/emo-bon/mgf-s3-prototype/blob/main/EMOBON00092/RNA-counts.dvc"},
-                        {"@id": "https://github.com/emo-bon/mgf-s3-prototype/blob/main/EMOBON00092/fastp.html.dvc"},
-                        {"@id": "https://github.com/emo-bon/mgf-s3-prototype/blob/main/EMOBON00092/final.contigs.fa.dvc"},
+                        {"@id": "EMOBON00092/DBH.merged.cmsearch.all.tblout.deoverlapped"},

```

Comment:
all these @id should become relative links starting from location of medata.json file after @base is corrected --> so would all simply start with `./ `